examples/printing-client/src/main/java/com/stellarstation/api/examples/printingclient/PrintingClientMain.java

```diff
@@ -0,0 +1,117 @@
+import com.stellarstation.api.v1.StellarStationServiceGrpc;
+import com.stellarstation.api.v1.StellarStationServiceGrpc.StellarStationServiceStub;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.auth.MoreCallCredentials;
+import io.grpc.stub.StreamObserver;
+import java.net.URI;
+import java.util.Base64;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PrintingClientMain {
+
+  private static final Logger logger = LoggerFactory.getLogger(PrintingClientMain.class);
+
+  public static void main(String[] args) throws Exception {
+    // Load the private key downloaded from the StellarStation Console.
+    ServiceAccountJwtAccessCredentials credentials =
+        ServiceAccountJwtAccessCredentials.fromStream(
+            Resources.getResource("api-key.json").openStream(),
+            URI.create("https://api.stellarstation.com"));
+
+    // Setup the gRPC client.
+    ManagedChannel channel =
+        ManagedChannelBuilder.forAddress("localhost", 8081)
+            // Only for testing, this should not be used when accessing the actual API
+            .usePlaintext()
+            .build();
+    StellarStationServiceStub client =
+        StellarStationServiceGrpc.newStub(channel)
+            .withCallCredentials(MoreCallCredentials.from(credentials));
+
+    // Open the stream with a listener to handle telemetry responses.
+    StreamObserver<SatelliteStreamRequest> requestStream =
+        client.openSatelliteStream(
+            new StreamObserver<>() {
+              @Override
+              public void onNext(SatelliteStreamResponse value) {
+                logger.info(
+                    "Got response: {}",
+                    Base64.getEncoder()
+                        .encodeToString(
+                            value
+                                .getReceiveTelemetryResponse()
+                                .getTelemetry()
+                                .getData()
+                                .toByteArray())
+                        .substring(0, 100));
+              }
+
+              @Override
+              public void onError(Throwable t) {
+                logger.warn("Got error from server.", t);
+              }
+
+              @Override
+              public void onCompleted() {}
+            });
+
+    // Send the first request to activate the stream.
+    requestStream.onNext(SatelliteStreamRequest.newBuilder().setSatelliteId("5").build());
+
+    ScheduledExecutorService commandExecutor = Executors.newScheduledThreadPool(1);
+
+    // Send some arbitrary commands every 3 seconds.
+    commandExecutor.scheduleAtFixedRate(
+        () -> {
+          requestStream.onNext(
+              SatelliteStreamRequest.newBuilder()
+                  .setSatelliteId("5")
```

Comment:
Yeah - I was thinking it's a good sanity check to leave it in every request since it's not really much overhead, and it reduces a "branch" within the API. What do you think?